Library/Homebrew/os/mac.rb

```diff
@@ -41,6 +41,10 @@ def cat
       version.to_sym
     end
 
+    def language
+      @language ||= `defaults read .GlobalPreferences AppleLanguages`.delete(" \n\"()").sub(%r{,.*}, '')
```

Comment:
Also is it worth using `gsub`? I'd also favour `//` regexes unless they contain `/`.